cms/static/sass/views/_certificates.scss

```diff
@@ -0,0 +1,723 @@
+// studio - views - certificates
+// ====================
+.view-certificates {
+
+  .content-primary, .content-supplementary {
+    @include box-sizing(border-box);
+    float: left;
+  }
+
+  .content-primary {
+    width: flex-grid(9, 12);
+    margin-right: flex-gutter();
+
+    .notice-moduledisabled {
+      @extend %ui-well;
+      @extend %t-copy-base;
+      background-color: $white;
+      padding: ($baseline*1.5) $baseline;
+      text-align: center;
+    }
+
+    .no-content {
+      @extend %no-content;
+    }
+
+    .wrapper-certificate {
+      margin-bottom: ($baseline*1.5);
+
+      .title {
+        @extend %t-title4;
+        @extend %t-strong;
+        margin-bottom: ($baseline/2);
+      }
+
+      .copy {
+        @extend %t-copy-sub1;
+      }
+    }
+
+    .wrapper-collection {
+      @extend %ui-window;
+      position: relative;
+      outline: none;
+
+      &:hover .collection .actions {
+        opacity: 1.0;
+      }
+
+      .collection-details {
+        padding: $baseline ($baseline*1.5);
+      }
+
+      .collection-header {
+        margin-bottom: 0;
+        border-bottom: 0;
+        padding-bottom: 0;
+
+        .title {
+          @extend %cont-truncated;
+          @extend %t-title5;
+          @extend %t-strong;
+          margin-right: ($baseline*14);
+          color: $black;
+
+          .toggle {
+            display: inline-block;
+            padding-left: $baseline;
+            color: $black;
+
+            &:hover, &:focus {
+              color: $blue;
+            }
+
+            .ui-toggle-expansion {
+              @include transition(rotate .15s ease-in-out .25s);
+              @extend %t-action1;
+              display: inline-block;
+              width: ($baseline*0.75);
+              vertical-align: baseline;
+              margin-left: -$baseline;
+            }
+
+            &.is-selectable {
+              @extend %ui-fake-link;
+
+              &:hover {
+                color: $blue;
+
+                .ui-toggle-expansion {
+                  color: $blue;
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .collection-info {
+        @extend %t-copy-sub1;
+        color: $gray-l1;
```

Comment:
remove this margin left and color rule (pairs with removal of expand / collapse action for this area.